docs/modules/operation/pages/deep-dive/notifications/commands.adoc

```diff
@@ -73,16 +105,13 @@ and not:
    -Dnodeid=8
 ----
 
-=== Valid switches from the NotificationManager class
+=== Valid switches from NotificationManager
```

Comment:
```suggestion
=== Valid switches from `NotificationManager`
```

Reply:
Do we want to use unicode formatting in headers? I think we had discussed it previously and decided against it, because of formatting. 🤔

Reply:
You are correct. Over editing on my part. :-) Please ignore!